api_v3/web/getFeed.php

```diff
@@ -73,6 +73,7 @@ function getRequestParameter($paramName)
 $entryId = getRequestParameter('entryId');
 $limit = getRequestParameter('limit');
 $ks = getRequestParameter('ks');
+$playerUrlType = getRequestParameter('playerUrlType');
```

Comment:
ok, I opened this pull request instead: https://github.com/kaltura/server/pull/4036